ckanext/unaids/react/components/FileInputComponent/src/HiddenFormInputs.js

```diff
@@ -2,6 +2,11 @@ import React from 'react';
 
 export default function HiddenFormInputs({ hiddenInputs }) {
 
+    hiddenInputs = {
+        ...hiddenInputs,
+        last_modified: new Date().toISOString().replace('Z', '')
```

Comment:
I think that the fact you need to remove tz info from the date string is a bug in ckanext-scheming.
Let's leave it for now (as it doesn't work otherwise) but I'll try to submit an issue.

Reply:
https://github.com/ckan/ckanext-scheming/pull/291